app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt

```diff
@@ -823,6 +823,10 @@ private val Event.wrapper: EventWrapper<*>?
             { RecentBookmarks.shown.record(it) }
         )
 
+        is Event.RecentBookmarkCount -> EventWrapper<NoExtraKeys>(
```

Comment:
This won't work.  Take a look at `HistoryRecentSearchesTapped` for when you have value with your event.